src/schemas/space.json

```diff
@@ -78,6 +78,12 @@
           "pattern": "^[A-Za-z0-9_-]*$",
           "maxLength": 39
         },
+        "farcaster": {
+          "type": "string",
+          "title": "farcaster",
+          "pattern": "^[a-z0-9][a-z0-9-]*$",
```

Comment:
I think we need to allow ens names as well, for example `alice.eth` as mentioned here https://docs.farcaster.xyz/learn/what-is-farcaster/usernames

Reply:
Removed the regex pattern to allow any string (we don't have validation in place for validating ENS name)